test/test_util/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- encoding: utf-8 -*-
+__author__ = 'feurerm'
```

Comment:
I don't think I authored this ;) Could you please drop that line?

Reply:
sorry, copy-paste problem